doc/pandas-2.0/source/goals.rst

```diff
@@ -0,0 +1,194 @@
+.. _goals:
+
+=======================
+ Goals and Motivations
+=======================
+
+.. note::
+
+  These documents are largely written by Wes McKinney, and at this point
+  reflect his opinions for the time being. Many things may change as we discuss
+  and work to reach a consensus about the path forward.
+
+The pandas codebase is now over 8 years old, having grown to over 200,000 lines
+of code from its original ~10,000 LOC in the original 0.1 open source release
+in January 2010.
+
+At a high level, the "pandas 2.0" effort is based on a number of observations:
+
+* The pandas 0.x series of releases have consisted with huge amounts of
```

Comment:
This first point muddles the waters a bit (for me); likewise with the point starting on line 79. " **Removal of deprecated / underutilized functionality**". 

Further down you seem to split between

1. API changes that are the result of pandas 0.X implementation details (e.g. fixing integer-NA), and
2. API changes that would be because the original idea may be flawed / out of scope (`.ix` `plotting`).

Does it make sense to limit this document *entirely* to the internals refactoring, and only talk about  API changes of the first kind?